packages/suite/src/components/suite/FormattedNftAmount.tsx

```diff
@@ -1,89 +1,127 @@
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-width: 145px;
-`;
+import { useTranslation } from 'src/hooks/suite';
+import { BlurUrls } from 'src/views/wallet/tokens/common/BlurUrls';
 
 export interface FormattedNftAmountProps {
     transfer: TokenTransfer;
     signValue?: SignValue;
     className?: string;
     isWithLink?: boolean;
+    alignMultitoken?: 'flex-end' | 'flex-start';
 }
 
 export const FormattedNftAmount = ({
     transfer,
     signValue,
     className,
     isWithLink,
+    alignMultitoken = 'flex-end',
 }: FormattedNftAmountProps) => {
-    const id = getNftTokenId(transfer);
-
+    const theme = useTheme();
+    const { translationString } = useTranslation();
     const { selectedAccount } = useSelector(state => state.wallet);
     const { network } = selectedAccount;
-    const explorerUrl =
-        network?.networkType === 'ethereum'
-            ? `${network?.explorer.nft}/${transfer.contract}/${id}`
-            : undefined;
 
-    const idComponent = <Id>{id}</Id>;
-    const symbolComponent = transfer.symbol ? <Symbol>&nbsp;{transfer.symbol}</Symbol> : null;
+    const symbolComponent = transfer.symbol ? (
+        <Text maxWidth={120} ellipsisLineCount={1}>
+            {transfer.symbol}
+        </Text>
+    ) : null;
+
+    const isMultitoken = isNftMultitokenTransfer(transfer);
+
+    if (isMultitoken) {
+        const tokens = transfer.multiTokenValues;
+
+        return (
+            <HiddenPlaceholder>
+                <Column alignItems={alignMultitoken}>
+                    {tokens?.map((token, index) => (
+                        <Row key={`${token.id}-${index}`} gap={spacings.xxs}>
+                            <Row>
+                                {signValue ? <Sign value={signValue} /> : null}
+                                {transfer.name ? (
+                                    <BlurUrls
+                                        text={translationString('TR_COLLECTION_NAME_OF_TOKEN_ID', {
+                                            tokenValue: token.value,
+                                            collectionName: transfer.name,
+                                        })}
+                                    />
+                                ) : (
+                                    <Row gap={spacings.xxs}>
+                                        <Row>{token.value}x</Row>
+                                        <Translation id="TR_TOKEN_ID" />
+                                    </Row>
+                                )}
+                            </Row>
+                            {isWithLink && network?.networkType === 'ethereum' ? (
+                                <TrezorLink
+                                    href={`${network?.explorer.nft}/${transfer.contract}/${token.id}`}
+                                    color={theme.textSecondaryHighlight}
+                                    variant="underline"
+                                    typographyStyle="label"
```

Comment:
It now does not have same font size in review tx modal 
![Screenshot 2024-11-29 at 17 48 11](https://github.com/user-attachments/assets/a66366ba-a54e-4abe-a46f-6736cb7ad05c)

Reply:
It's not the same on develop either

<img width="495" alt="image" src="https://github.com/user-attachments/assets/b7cd28eb-eb04-49f3-acf8-3fc6e84d4824">

Reply:
I also noticed that the color of amounts in the tx modal on develop is different, will adjust that